x-pack/plugins/maps/public/classes/layers/vector_layer/blended_vector_layer/blended_vector_layer.ts

```diff
@@ -57,6 +58,7 @@ function getClusterSource(documentSource: IESSource, documentStyle: IVectorStyle
     indexPatternId: documentSource.getIndexPatternId(),
     geoField: documentSource.getGeoFieldName(),
     requestType: RENDER_AS.POINT,
+    resolution: GRID_RESOLUTION.COARSE,
```

Comment:
This does not alter the behavior of BlendedVectorLayer

ESGeoGridSource.createDescriptor automatically sets resolution to GRID_RESOLUTION.COARSE when its not provided in Partial<ESGeoGridSourceDescriptor>. I wanted to make sure that the resolution is explicitly set in BlendedVectorLayer instead of relying on default behavior.